subsys/logging/Kconfig

```diff
@@ -192,6 +192,12 @@ config LOG_STRDUP_BUF_COUNT
 	  Each entry takes CONFIG_LOG_STRDUP_MAX_STRING bytes of memory plus
 	  some additional fixed overhead.
 
+config LOG_STRDUP_POOL_PROFILING
```

Comment:
Could we enable shell when somebody will select this option?

Reply:
I wouldn't do that. `log_get_strdup_pool_utilization` and `log_get_strdup_longest_string` are in the api so user may want to add log or printk and not use shell (e.g. low memory footprint).